lib/order.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'menu'
+require_relative 'takeaway'
```

Comment:
I'm assuming this is for further development?

Reply:
Yeah I had a go at changing a few things but reverted back to this structure. Left those in.